packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolygonController.m

```diff
@@ -112,7 +99,7 @@ - (void)addPolygons:(NSArray<FGMPlatformPolygon *> *)polygonsToAdd {
         [[FLTGoogleMapPolygonController alloc] initWithPath:path
                                                  identifier:identifier
                                                     mapView:self.mapView];
-    [controller updateFromPlatformPolygon:polygon registrar:self.registrar];
+    [controller updateFromPlatformPolygon:polygon];
```

Comment:
The registrar was either copypasta from another object, or it used to be needed and now isn't; it was unused, so I removed it in the refactoring.